cmd/hamctl/command/describe.go

```diff
@@ -74,7 +101,7 @@ Format the output with a custom template:
 			if len(template) == 0 {
 				template = describeReleaseDefaultTemplate
 			}
-			err = templateOutput(os.Stdout, "describeRelease", template, releasesResponse)
+			err = templateDescribeRelease(os.Stdout, template, mapReleaseResponseToTemplate(releasesResponse))
```

Comment:
A curious question (unrelated to the changes in this PR, but this is the first time I see the code) I can see how a making it possible to use a custom template makes it very flexible, but it also couples the clients to the DTO defined here. It might be very stable so maybe not a problem but I think we should add a comment that it is in fact an interface, and thus shouldn't be changed in a breaking way. Also, how does a client know what that interface looks like (i.e. what he can use in his custom template), he would have to go look at the code here? It should probably be included in the help description.

Reply:
Yes, that is a "classic" with templated commands, no one ever knows what the datamodel for it is.

A better help description would be nice to have and this change makes that very essy and way more stable. I'll see if I can follow up with that.